core/services/privacy_policy/Manager.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace EventEspresso\core\services\privacy_policy;
+
+use EventEspresso\core\exceptions\InvalidClassException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\exceptions\InvalidFilePathException;
+use EventEspresso\core\exceptions\InvalidIdentifierException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\services\collections\CollectionDetails;
+use EventEspresso\core\services\collections\CollectionInterface;
+use EventEspresso\core\services\collections\CollectionLoader;
+use WP_Post;
+
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
+
+
+
+/**
+ * Class PrivacyPolicyManager
+ * Manages setting up the hooks to add the EE core and add-ons' privacy policies
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ */
+class Manager
```

Comment:
I know that `privacy_policy` is in the namespace, but can we plz name this `PrivacyPolicyManager` so that it is always explicitly clear, so we can avoid conversations like this:

> Mr A : If you look in the Manager class you'll see such n such...
> Mr B : Which Manager class?
> Mr A : `\core\services\privacy_policy\Manager`
> Mr B : ahhh kk... thnx... uhhh...  I don't see "such n such" anywhere !?!?
> Mr A : Oh ya... that wasn't supposed to represent any actual code or text and was only added as a literary device to move this hypothetical conversation along in order to get to its real purpose.
> Mr B : Hypothetical conversation? Is... is none of this real???
> Mr A : Ya I'm sorry to inform you Mr B, but you are nothing more than an imaginary construct!
> Mr B : <existential crisis>

Reply:
agreed, let's not go down that road 🙃